pyccel/errors/errors.py

```diff
@@ -84,7 +85,8 @@ def __init__(self, *, stage, filename,
                  severity=None,
                  message='',
                  symbol=None,
-                 blocker=False):
+                 blocker=False,
+                 traceback=None):
```

Comment:
won't traceback default value  as ``` '' ```  better than None, so you can avoid check here ?
```   'traceback': '{}\n'.format(self.traceback) if self.traceback else '' ```

Reply:
I'm not sure if this helps as I don't want to add the line break if there is no traceback

Reply:
Actually the `\n` doesn't need to be there, but we still need to pass something from `Errors.report`